tokio/src/runtime/tests/task.rs

```diff
@@ -78,6 +78,44 @@ fn create_drop2() {
     handle.assert_dropped();
 }
 
+#[test]
+fn drop_abort_handle1() {
```

Comment:
These tests could be marked as `#[cfg(tokio_unstable)]`. e.g. https://github.com/tokio-rs/tokio/pull/4518/files#diff-88b4b8cbf7c654d7178a01ca42e7b48ceb9810d4309a0111aa3a5753e3a08e4cR521